rest-api/etl/export_etl_results.sh

```diff
@@ -22,7 +22,10 @@ fi
 PROJECT_AND_ACCOUNT=
 if [ "${PROJECT}" ]
 then
-  PROJECT_AND_ACCOUNT="--project ${PROJECT} --account ${ACCOUNT} --instance_name ${INSTANCE_NAME} --service_account exporter@${PROJECT}.iam.gserviceaccount.com"
+  PROJECT_AND_ACCOUNT="--project ${PROJECT} --account ${ACCOUNT} --service_account exporter@${PROJECT}.iam.gserviceaccount.com"
+fi
+if [ -n "${INSTANCE_NAME}" ]; then
```

Comment:
This is fine but you'd never use INSTANCE_NAME without also using PROJECT?

Reply:
(So I guess this could be inside the former if statement)

Reply:
Done.

What does it actually mean to omit project/account. Does that send against local?

Reply:
Yep.